src/osg/TextureRectangle.cpp

```diff
@@ -231,24 +231,19 @@ void TextureRectangle::apply(State& state) const
     }
     else if (_image.valid() && _image->data())
     {
-        GLExtensions * extensions = state.get<GLExtensions>();
```

Comment:
think this whole file can go back to the 3.6 version now that selectSizedInternalFormat is going back into Texture.cpp